spec/models/concerns/request_session_persistable_spec.rb

```diff
@@ -23,6 +24,32 @@ def reset_unused_fields
   let(:request_session) { {} }
   let(:current_user) { nil }
 
+  describe "#initialize" do
+    context "with a datetime attribute" do
+      let(:attributes) { { datetime: "2025-05-21 11:48:17 +0100" } }
+
+      it "isn't marked as having changed" do
+        expect(model.changed?).to be(false)
+      end
+
+      it "parses and keeps the time zone" do
+        expect(model.datetime).to eq(Time.zone.local(2025, 5, 21, 11, 48, 17))
+        expect(model.datetime).to be_a(ActiveSupport::TimeWithZone)
+        expect(model.datetime.time_zone.name).to eq("London")
+      end
+
+      context "when system timezone is UTC" do
```

Comment:
When I removed the `around` block, the example still passed. I guess `ClimateControl.modify` is not affecting the timezone configuration.

Reply:
Hmm I was worried that might be the case, it seems like the Ruby process needs to restart to pick up the latest `TZ`. In GitHub Actions we run the tests in UTC `TZ` so we are getting coverage, but it makes it tricky to test locally.